package.json

```diff
@@ -6,6 +6,7 @@
     "start": "rm -rf ./node_modules/.cache/babel-loader && node scripts/start.js",
     "start:holofuel": "REACT_APP_HOLOFUEL_APP=true npm start",
     "build": "node scripts/build.js",
+    "build:holofuel": "REACT_APP_HOLOFUEL_APP=true npm run build",
```

Comment:
does this work? And why is it in this PR?

Reply:
Good catch; somehow crept in there when creating a new branch.  This build approach *does* work btw, but I did not mean for this branch to include that change. I'll remove it.